pkg/yqutil/yqutil_test.go

```diff
@@ -0,0 +1,28 @@
+package yqutil
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestEvaluateExpression(t *testing.T) {
+	expression := `.cpus = 2 | .memory = "2GiB"`
```

Comment:
Can we also test a complex expression like mounts?

Reply:
Sure, probably should do an array like the template tests...